### PR TITLE
[SIP-6] removed get_data for BigNumber

### PR DESCRIPTION
**Pull request by conglei**
This PR is to move `get_data` logic for `BigNumber` and `BigNumber` Total into frontend code. 

#5692 

@kristw @williaster @mistercrunch

### CODE REVIEW

File: superset/assets/src/visualizations/BigNumber/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import adapter from './adaptor';
```
Comment (kristw) [inline superset/assets/src/visualizations/BigNumber/index.js]: typo `adapt"o"r`

File: superset/assets/src/visualizations/BigNumber/BigNumber.jsx [hunk 2]
```diff
@@ -26,7 +23,7 @@ const PROPORTION = {
   TRENDLINE: 0.3,
 };
 
-function renderTooltipFactory(formatValue) {
+export function renderTooltipFactory(formatValue) {
```
Comment (kristw) [inline superset/assets/src/visualizations/BigNumber/BigNumber.jsx]: This was used only in this file. Not sure it is useful to export now. (ok to expert if there are other places using or unit test)

Comment (conglei) [reply superset/assets/src/visualizations/BigNumber/BigNumber.jsx]: Actually it is also used in adaptor for the prop `renderTooltip`. That is why I exported it :)

Comment (kristw) [reply superset/assets/src/visualizations/BigNumber/BigNumber.jsx]: I missed that. Thank you!

**Pull request merged**